doc/geany.txt

```diff
@@ -1689,12 +1689,13 @@ Global tags files can have three different formats:
 * Pipe-separated format
 * CTags format
 
-The first line of global tags files should be a comment, introduced
-by ``#`` followed by a space and a string like ``format=pipe``,
-``format=ctags`` or ``format=tagmanager`` respectively, these are
+For the tagmanager or pipe-separated format, the first line of global tag files
+should be a comment, introduced by ``#`` followed by a space and a string like
```

Comment:
delete "a string like", its not "like" it is exactly one or other IIUC

Reply:
Will do (also not mine ;-).